pom.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.epam.tc</groupId>
+    <artifactId>sofiia_kalishchuk</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    </properties>
+
+    <profiles>
+        <profile>
+            <id>SumandSub</id>
+<!--            <activation>-->
+<!--                <activeByDefault>true</activeByDefault>-->
+<!--            </activation>-->
+            <properties>
+                <suite.xml.file>SumAndSub.xml</suite.xml.file>
+            </properties>
+        </profile>
+        <profile>
+            <id>All</id>
+            <properties>
+                <suite.xml.file>All.xml</suite.xml.file>
+            </properties>
+        </profile>
+        <profile>
+            <id>MultandDiv</id>
+            <properties>
+                <suite.xml.file>MultAndDiv.xml</suite.xml.file>
+            </properties>
+        </profile>
+    </profiles>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+            <version>7.6.1</version>
+        </dependency>
+        <dependency>
+            <groupId>repository.com.epam.tat.module4</groupId>
+            <artifactId>calculator</artifactId>
+            <version>1.0</version>
+            <scope>system</scope>
```

Comment:
It's not a good idea to use absolute paths in your project. Nobody will be able to run this. So, unfortunately,  it's impossible to check your HW now.
You'd better place this jar file somewhere in your project - in separate directory (e.g., resources) - and then provide a relative path.
So please, fix this issue first.